verifact/forum/schema.py

```diff
@@ -55,19 +47,30 @@ class Meta:
         interfaces = (relay.Node,)
 
 
-class QuestionConnection(relay.Connection):
+class VoteConnection(relay.Connection):
     class Meta:
-        node = QuestionNode
+        node = VoteNode
 
 
-class AnswerConnection(relay.Connection):
+class AnswerNode(DjangoObjectType):
     class Meta:
-        node = AnswerNode
+        model = Answer
+        interfaces = (relay.Node,)
+        convert_choices_to_enum = False
 
+    viewer_vote = ConnectionField(VoteConnection)
+    citations = ConnectionField(CitationConnection)
 
-class VoteConnection(relay.Connection):
+    def resolve_viewer_vote(self,args):
+        return Vote.objects.filter(answer=self).filter(user=args.context.user)
```

Comment:
Since you are searching for a single vote, it would be cleaner to either use `get` like `Vote.objects.get(answer=self, user=args.context.user)` (while handling the `DoesNotExist` exception), or to include both conditions in a single call to filter. I would vote for `get` as it's more semantically correct. See [Django docs on get](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#get).